frontend/src/components/Home/SearchBar.tsx

```diff
@@ -1,17 +1,17 @@
-import React, {ReactElement} from 'react';
+import React, { ReactElement } from 'react';
 
-  const SearchBar=(): ReactElement => {
-    return <div className="search input-group mb-3">
-    <input type="text" className="form-control" placeholder="Search by any location..." aria-label="Search locations" aria-describedby="basic-addon2"/>
+const SearchBar = (): ReactElement => {
+  return <div className="search input-group mb-3">
+    <input type="text" className="form-control" placeholder="Search by any location..." aria-label="Search locations" aria-describedby="basic-addon2" />
     <div className="input-group-append">
       <button className="btn btn-light" type="button">
-      <svg width="1em" height="1em" viewBox="0 0 16 16" className="bi bi-search" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
-      <path fillRule="evenodd" d="M10.442 10.442a1 1 0 0 1 1.415 0l3.85 3.85a1 1 0 0 1-1.414 1.415l-3.85-3.85a1 1 0 0 1 0-1.415z"/>
-      <path fillRule="evenodd" d="M6.5 12a5.5 5.5 0 1 0 0-11 5.5 5.5 0 0 0 0 11zM13 6.5a6.5 6.5 0 1 1-13 0 6.5 6.5 0 0 1 13 0z"/>
-    </svg>
+        <svg width="1em" height="1em" viewBox="0 0 16 16" className="bi bi-search" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Also mentioned this in Kinjal's PR, but instead of pasting SVG data into the React frontend code, you should put it in an `.svg` file (maybe make an `assets/` folder in the frontend src) and then import it to use it as a component.